pkg/apis/greenhouse/v1alpha1/cluster_types.go

```diff
@@ -4,13 +4,19 @@
 package v1alpha1
 
 import (
+	"time"
+
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 // ClusterSpec defines the desired state of the Cluster.
 type ClusterSpec struct {
 	// AccessMode configures how the cluster is accessed from the Greenhouse operator.
 	AccessMode ClusterAccessMode `json:"accessMode"`
+
+	// MaxTokenValidity specifies the maximum duration for which a token remains valid
+	// +kubebuilder:default:="72h"
+	MaxTokenValidity metav1.Duration `json:"maxTokenValidity,omitempty"`
```

Comment:
I think we should add some structure here by using a custom struct field`KubeConfig` to contain the `MaxTokenValidity`. This way we could also add other KubeConfig specific configuration such as Token secret reference in the feature. 
@kengou what do you think?

Reply:
That is a good idea

Reply:
Done